lib/openapi-connector.js

```diff
@@ -129,6 +129,27 @@ OpenApiConnector.prototype.connect = function(cb) {
           responseInterceptor: self.connectorHooks.afterExecute,
         };
 
+        if (req.spec.openapi) {
+          let baseURL;
+          if (self.settings.spec && typeof self.settings.spec === 'string') {
+            if (self.settings.spec.includes('http') || self.settings.spec.includes('https')) {
```

Comment:
Can we explicitly check if it's at the start?

```suggestion
            if (self.settings.spec.indexOf('http') > 0 || self.settings.spec.indexOf('https') > 0) {
```

Reply:
I think you meant the condition to be like this: `self.settings.spec.indexOf('http') === 0 || self.settings.spec.indexOf('https') === 0`.

Reply:
Sorry for my limited understanding but I can't see why we should explicitly check if http is in the start.